src/CoreEditor/webrunesAPI.js

```diff
@@ -1,42 +0,0 @@
-/**
```

Comment:
Why it's deleted? Maybe we don't need most of the functions now (btw, not sure about all) but we will need them later. Comment off unnecessary now.